fondant/compiler.py

```diff
@@ -126,10 +126,17 @@ def _generate_spec(self, pipeline: Pipeline) -> dict:
             volumes = [asdict(volume)] if volume else []
 
             services[safe_component_name] = {
-                "image": component_op.component_spec.image,
                 "command": command,
                 "depends_on": depends_on,
                 "volumes": volumes,
             }
 
+            if component_op.local_component:
+                services[safe_component_name][
+                    "build"
+                ] = f"./{Path(component_op.component_spec_path).parent}"
```

Comment:
Can the `output_path` argument be any location on your filesystem? If so, does it make sense to use the "./{Path()}" format or should make it depend on the `output_path`? I assume the `docker-compose up` will be ran from the `output_path`, which could be in a different directory than the local components.

Reply:
You mean reference the component code relative from the output_path ?